compiler/cpp/src/thrift/generate/t_php_generator.cc

```diff
@@ -545,30 +553,41 @@ void t_php_generator::generate_consts(vector<t_const*> consts) {
     f_consts << "final class Constant extends \\Thrift\\Type\\TConstant {" << endl;
 
     indent_up();
+    if (php56_) {
+      // Create const property
+      for (c_iter = consts.begin(); c_iter != consts.end(); ++c_iter) {
+        string name = (*c_iter)->get_name();
+
+        indent(f_consts) << "public CONST $" << name << " = ";
+        generate_php_doc(f_consts, *c_iter);
+        f_consts << render_const_value((*c_iter)->get_type(), (*c_iter)->get_value());
+        f_consts << ";" << endl;
+      }
+    } else {
+      // Create static property
```

Comment:
Is it worth maintaining the static property for backwards compatibility. (5.6+ users could opt into the const without having to update all callers).